src/django_pg_migration_tools/indexes.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+from django.db import models
+from django.db.backends import ddl_references
+from django.db.backends.base import schema as base_schema
+from django.db.models.indexes import Index
+
+
+class UniqueIndex(Index):
+    def create_sql(
+        self,
+        model: type[models.Model],
+        schema_editor: base_schema.BaseDatabaseSchemaEditor,
+        using: str = "",
+        **kwargs: Any,
```

Comment:
🐼 in this case you're not accessing anything from the kwargs and so you can use `object` instead.

Reply:
Perfect: dcadb0dd61e7d086b229b7952c071ccb1c9ea5ee